app/models/govuk_publishing_components/component_docs.rb

```diff
@@ -34,6 +38,10 @@ def component_in_use(component)
       true if @limit_to.include?(component)
     end
 
+    def component_not_in_use(component)
+      true if @limit_to.exclude?(component)
+    end
+
```

Comment:
Again similar to above maybe we could combine `component_in_use` and `component__not_in_use`? like this:

```
    def component_in_use?(component, in_use:)
      in_use ? @limit_to.include?(component) : @limit_to.exclude?(component)
    end
```

Again no problem if it's too confusing!